e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide to Running Tests
+
+It is important to note that one must ensure that all dfusion related builds are recent (i.e. driver, graph-listener and their dependents). 
+
+From within the project source run:
+
+```shell script
```

Comment:
ubernit:
```suggestion
```sh
```

Reply:
Ya... My IDE auto completes to this for some stupid reason that I will have to investigate.